kotlin/build.gradle.kts

```
@@ -0,0 +1,31 @@
+// In this section you declare the dependencies for your production and test code
+dependencies {
+    val versionKotlin = "1.6.0"
+
+    // Project
+    compileOnly(project(":core"))
+
+    // Kotlin dependencies
+    api(group = "org.jetbrains.kotlinx", name = "kotlinx-coroutines-core", version = versionKotlin)
+
+    // Tests
+    testImplementation(project(":core"))
+    testImplementation(project(":handler-simple"))
+    testImplementation(testFixtures(project(":core")))
+    testImplementation(group = "org.jetbrains.kotlinx", name="kotlinx-coroutines-test", version = versionKotlin)
```

I think we only need a dependency on the api module for kotlin, there is no need for compileOnly unless i'm missing something.

```suggestion
    // project
    api(project(":api"))
    testImplementation(project(":core"))
    testImplementation(project(":handler-simple"))
    testImplementation(testFixtures(project(":core")))

    // kotlin
    api(group = "org.jetbrains.kotlinx", name = "kotlinx-coroutines-core", version = versionKotlin)
    testImplementation(group = "org.jetbrains.kotlinx", name="kotlinx-coroutines-test", version = versionKotlin)
```

---

Good call!

---

I also inlined versionKotlin. Since we're not sure if dependabot picks up variables